execution/executor/src/metrics.rs

```diff
@@ -53,12 +53,12 @@ pub static COMMIT_CHUNK: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
-pub static VM_EXECUTE_BLOCK: Lazy<Histogram> = Lazy::new(|| {
+pub static BLOCK_EXECUTOR_EXECUTE_BLOCK: Lazy<Histogram> = Lazy::new(|| {
     register_histogram!(
         // metric name
-        "aptos_executor_vm_execute_block_seconds",
+        "aptos_executor_block_executor_execute_block_seconds",
         // metric description
-        "The time spent in seconds of vm block execution in Aptos executor",
+        "The time spent in seconds of BlockExecutor block execution in Aptos executor",
```

Comment:
This seems duplicated with what we have in `counters.rs`?. The comment surely is (and is also not clear)